entgql/internal/todo/ent.resolvers.go

```diff
@@ -17,7 +17,47 @@ package todo
 // This file will be automatically regenerated based on the schema, any resolver implementations
 // will be copied through when generating and any unknown code will be moved to the end.
 
+import (
```

Comment:
Isn't this file a duplication of todo.resolvers.go? It's because the configuration in gqlgen is "follow-schema". Either rename the generated GrpahQL schema to "todo.graphql" or delete https://github.com/ent/contrib/blob/master/entgql/internal/todo/todo.resolvers.go.

Reply:
The todo.resolvers.go still have resolvers for todo's mutation. So, I don't think I have to change anything.

https://github.com/ent/contrib/blob/giautm/update-todo-example/entgql/internal/todo/todo.resolvers.go